deis/azuredeploy.json

```diff
@@ -0,0 +1,211 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "String",
+            "defaultValue": "West US",
+            "metadata": { 
+                 "description": "Location of resources" 
+            } 
+        },
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": { 
+                 "description": "Name of the of the storage account for VM OS Disk"
+            } 
+        },
+        "vmNamePrefix": {
+            "type": "string",
+            "defaultValue": "coreos",
+            "metadata": { 
+                 "description": "Prefix for the names of the VMs"
+            } 
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "defaultValue": "myVNET",
+            "metadata": { 
+                 "description": "Name of the VNET hosting the VMs"
+            } 
+        },
+        "vmSourceImageName": {
```

Comment:
This format isn't recommended anymore. Use the Pub/Offer/SKU/Version method for images https://github.com/Azure/azure-quickstart-templates/blob/master/breaking-change.md. You can get the entire set of images using CLI or PS http://azure.microsoft.com/en-us/documentation/articles/resource-groups-vm-searching/